### DISCUSSION TITLE
Internet Radio Support

### DISCUSSION THREAD

**Discussion by R1D3R175** [MissingCore/Music#43] [Ideas]
Would love being able to tune to some of my favorite SomaFM's radio stations :)

**Comment by cyanChill** [comment]
~~I haven't look into it too deeply, but this could be implemented with https://github.com/dehy/react-native-radio-player. Since I primarily want to focus this app as a music player, this will be put on the backburner, but I might take a look back at this in the future.~~

**EDIT:** I'm switching this to "Not Planned" as I want to keep this app strictly as a music player.
